tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/AdoNetDefinitions.cs

```diff
@@ -3,56 +3,86 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet;
+using Datadog.Trace.Configuration;
 using static Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.AdoNetClientInstrumentMethodAttribute;
 using static Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.AdoNetConstants;
 
-/********************************************************************************
- * Task<int> .ExecuteNonQueryAsync(CancellationToken)
- ********************************************************************************/
+#pragma warning disable SA1118 // parameter spans multiple lines
+[assembly: AdoNetClientInstrumentMethod(
```

Comment:
Overall I really like design of this, it's easy to see for a given assembly/version/type, which methods we want to instrument. I have two small suggestions on naming:
- What if we rename `AdoNetClientInstrumentMethod` to `AdoNetClientInstrumentMethods` (additional "s" at the end)? This could help indicate that this represents a collection of methods. Or we could even append `Collection` to make it even more clear. What do you think?
- What do you think of renaming `SignatureAttributes` to `TargetMethodAttributes`? The new name seems slightly clearer to me, but I have no objection to the original name

Reply:
Fine by me - renamed in https://github.com/DataDog/dd-trace-dotnet/pull/2288/commits/bb06fe0e6040cc2707f475d33e03a3e300719ddf